src/status/StatusItem.cpp

```diff
@@ -27,6 +27,12 @@ StatusItem::StatusItem(const std::string &name, const type::Type type) :
 	case type::INT:
 		_value = 0;
 		break;
+	case type::BYTE:
+		_value = (unsigned char)0;
+		break;
+	case type::SHORT:
+		_value = (unsigned short)0;
```

Comment:
Yeah, that's a bit odd. this are status items and they can be different types like EPICS channels so we have a field to store the type and the actual value as a `boost` `any`. Every time we reach for the value we must check the type

BYTE and SHORT were not considered so it was giving a warning about non exhaustive `switch`